modules/govuk/manifests/app/config.pp

```diff
@@ -415,14 +415,6 @@
   }
 
   if $health_check_path != 'NOTSET' {
-    @@icinga::check { "check_app_${title}_up_on_${::hostname}":
-      ensure              => $ensure,
-      check_command       => "check_app_health!check_app_up!${port} ${health_check_path}",
-      service_description => "${title} app healthcheck",
-      host_name           => $::fqdn,
-      notes_url           => monitoring_docs_url(app-healthcheck-failed),
-      contact_groups      => $additional_check_contact_groups,
-    }
```

Comment:
With this change, there won't be any check for apps where `$json_health_check` is `false` if that makes sense, which is bad as having some form of healthcheck for each app is useful.

Reply:
Ah, I appear to have misunderstood the task. 😳I thought that every app had both of them, just we noticed it in email-alert-api first. It does indeed [appear that not all apps have `json_health_check` set to true](https://github.com/alphagov/govuk-puppet/search?q=json_health_check&unscoped_q=json_health_check). I wonder why not.

In light of this, should we:

- Set `json_health_check` to true by default (ie for every app) and do the work to all the apps to make that work (disclaimer: I don't know what that work would involve).
- Edit the `if $json_health_check` conditional to put the "app_name app healthcheck" in the `else` part. This would do the thing we want.

I'm leaning towards the latter.

Reply:
I went with the `else` option, with some re-wording of the commit message, in e8f631e5af51665292bd67e7e36ea0dd4ece626d.

Reply:
:+1: